packages/web3-eth/src/accounts.js

```diff
@@ -111,11 +111,99 @@ function recover (hash, signature) {
     return EthFP.Account.recover(hash, signature);
 }
 
+function encrypt (account, password) {
+    // TODO: implement
+    return JSON.stringify(account);
+}
+
+function decrypt (account, password) {
+    return JSON.parse(account);
+}
+
+// Note: this is trying to follow closely the specs on
+// http://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html
+
+function Wallet() {
+    this.length = 0;
+}
+
+Wallet.prototype.create = function (numberOfAccounts, entropy) {
```

Comment:
Im unsure, if we shouldn't rename it to `generate` or so, as you can add new accounts with this function.